Cu 86865210e matchup scaffolding (do last)

brownKasey opened the pull request.

brownKasey committed:
#868651z6b[in development] added basic scaffholding

brownKasey committed:
#868651z6b[in development] added basic scaffholding

brownKasey committed:
#868651z6b[in development] added basic scaffholding

brownKasey committed:
#86865210e[in development] added basic scaffholding

MrTofuuu commented:
Task linked: [CU-86865210e Matchup Scaffolding](https://app.clickup.com/t/86865210e)

MrTofuuu committed:
Delete client/src/pages/Profile.jsx

Removing the profile page bc I think you already created that in the other `profile` branch?

MrTofuuu reviewed (approved):
LTGM
